scripts/docuCheck.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e # exit on errors
+
+BASE_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+declare -a LUIGI_FOLDERS=(
+  "client"
+  "core"
+)
+
+source $BASE_DIR/shared/bashHelpers.sh
+
+installPrerequisites() {
```

Comment:
I would decoupled completely Luigi and Core documentation in this step. Fo that I would run the complete logic of this function once for Client and once for Core, to prevent difficult to find bugs in the future due to coupling created among Luigi and Core documentation. What do you think?